https://issues.redhat.com/browse/ACM-6594 Cleaning up GRC for 2.9 (#5972)
open-cluster-management/rhacm-docs: governance/cert_policy_ctrl.adoc

[#certificate-policy-controller]
= Certificate policy controller

> You can use the certificate policy controller to detect certificates that are close to expiring, time durations (hours) that are too long, or contain DNS names that fail to match specified patterns. You can add the certificate policy to the `policy-templates` field of the policy on the hub cluster, and propagate it to the selected managed clusters by using the governance framework. See the xref:../governance/policy_overview.adoc#policy-overview[Policy overview] documentation for more details on the hub cluster policy.

Author: ```suggestion
You can use the certificate policy controller to detect certificates that are close to expiring, time durations (hours) that are too long, or contain DNS names that fail to match specified patterns. You can add the certificate policy to the `policy-templates` field of the policy on the hub cluster, which propagates to the selected managed clusters by using the governance framework. See the xref:../governance/policy_overview.adoc#policy-overview[Policy overview] documentation for more details on the hub cluster policy.
```
